manageiq-operator/deploy/olm-catalog/manageiq-operator/0.0.1/manageiq-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -58,7 +58,7 @@ spec:
                 - name: OPERATOR_NAME
                   value: manageiq-operator
                 image: docker.io/manageiq/manageiq-operator:v0.0.1
-                imagePullPolicy: Always
+                imagePullPolicy: IfNotPresent
```

Comment:
Why are we switching from `Always` to `IfNotPresent`?

Reply:
I think whenever you're using a specific tag it should probably not be `Always`.

Also `Always` here wasn't a conscious decision, I believe it was just the default from the `operator-sdk new`